app/views/admin/merchants/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Create Merchant</h1>
+<%= form_with model: @merchant, url: "/admin/merchants", method: :post, local: true do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :name %>
+
+  <%= f.submit %>
+  
```

Comment:
I think this will still read fine with the whitespace removed here, but that can happen later.